Modules/timemodule.c

```diff
@@ -1026,11 +1026,11 @@ time_ctime(PyObject *self, PyObject *args)
 }
 
 PyDoc_STRVAR(ctime_doc,
-"ctime(seconds) -> string\n\
+"ctime([seconds]) -> string\n\
 \n\
 Convert a time in seconds since the Epoch to a string in local time.\n\
-This is equivalent to asctime(localtime(seconds)). When the time tuple is\n\
-not present, current time as returned by localtime() is used.");
+This is equivalent to asctime(localtime(seconds)). When 'seconds' is not\n\
+passed in, the current time as returned by time() is used.");
```

Comment:
According to https://docs.python.org/3/library/time.html#time.ctime, it should be 
```suggestion
This is equivalent to asctime(localtime(seconds)). When 'seconds' is not\n\
specified or None, the current time as returned by time() is used.");
```

Reply:
Good catch on the None option. I intended to correct the ctime() doc entry by matching the language found in the docs for localtime() and gmtime(), which can also take None as an argument but make no mention of it.

Reply:
The docstring should be similar to the doc https://docs.python.org/dev/library/time.html#time.ctime which says:

> If secs is not provided or None, the current time as returned by time() is used.